packages/roosterjs-editor-dom/lib/utils/getTableFormatInfo.ts

```diff
@@ -0,0 +1,81 @@
+import { TableFormat } from 'roosterjs-editor-types';
+const TABLE_STYLE_INFO = 'roosterTableInfo';
+
+/**
+ * Get the format info of a table
+ * If the table does not have a info saved, it will be retrieved from the css styles
+ * @param table The table that has the info
+ */
+export default function getTableFormatInfo(table: HTMLTableElement) {
+    const obj = safeParseJSON(table?.dataset[TABLE_STYLE_INFO]) as TableFormat;
+    return checkIfTableFormatIsValid ? obj : undefined;
+}
+
+function checkIfTableFormatIsValid(format: TableFormat) {
+    const {
+        topBorderColor,
+        verticalBorderColor,
+        bottomBorderColor,
+        bgColorOdd,
+        bgColorEven,
+        hasBandedColumns,
+        hasBandedRows,
+        hasFirstColumn,
+        hasHeaderRow,
+        tableBorderFormat,
+    } = format;
+    const colorsValues = [
+        ...Object.values({
```

Comment:
Object.values is not supported by Chrome 49 and IE.
<img width="739" alt="image" src="https://user-images.githubusercontent.com/23065085/151106867-11244dc2-a868-4c55-a44d-372c0557d96c.png">